src/commands/deploy/deploy.ts

```diff
@@ -353,13 +354,14 @@ const uploadDeployBlobs = async ({
   silent,
   siteId,
 }: {
-  cachedConfig: any
+  cachedConfig: $TSFixMe
   deployId: string
   options: OptionValues
   packagePath?: string
   silent: boolean
   siteId: string
 }) => {
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
You could use this: https://github.com/netlify/cli/blob/e1372a3c228246c7feb9bf4f4e63dad547b2bfa9/src/utils/command-helpers.ts#L300

Reply:
TBH It feels to me awkward to have a noop function definition to bypass a useless lint rule

Reply:
Fair, but in that case we should disable the lint rule as a whole rather than adding exceptions on every instance?

Reply:
Done